client/landing/jetpack-cloud/sections/backups/rewind-flow/restore.tsx

```diff
@@ -1,15 +1,206 @@
+
+	const restoreTimestamp = moment.unix( rewindId ).format( 'LLL' );
+
+	const inProgressRewindStatus = useSelector( state =>
+		getInProgressRewindStatus( state, siteId, rewindId )
+	);
+	const inProgressRewindPercentComplete = useSelector( state =>
+		getInProgressRewindPercentComplete( state, siteId, rewindId )
+	);
+
+	const requestRestore = useCallback(
+		() => dispatch( rewindRestore( siteId, rewindId, rewindConfig ) ),
+		[ dispatch, rewindConfig, rewindId, siteId ]
+	);
+
+	const onConfirm = () => {
+		setUserHasRequestedRestore( true );
+		requestRestore();
+	};
+
+	const renderConfirm = () => (
+		<>
+			<div className="rewind-flow__header">
+				<Gridicon icon="history" size={ 48 } />
+			</div>
+			<h3 className="rewind-flow__title">{ translate( 'Restore site' ) }</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'{{strong}}%(restoreTimestamp)s{{/strong}} is the selected point for your restore. ',
+					{
+						args: {
+							restoreTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<h4 className="rewind-flow__cta">{ translate( 'Choose the items you wish to restore:' ) }</h4>
+			<RewindConfigEditor currentConfig={ rewindConfig } onConfigChange={ setRewindConfig } />
+			<RewindFlowNotice
+				gridicon="notice"
+				title={ translate( 'This will override and remove all content after this point' ) }
+				type={ RewindFlowNoticeLevel.WARNING }
+			/>
+			<Button
+				className="rewind-flow__primary-button"
+				primary
+				onClick={ onConfirm }
+				disabled={ Object.values( rewindConfig ).every( setting => ! setting ) }
+			>
+				{ translate( 'Confirm restore' ) }
+			</Button>
+		</>
+	);
+
+	const renderInProgress = () => (
+		<>
+			<div className="rewind-flow__header">
+				<Gridicon icon="history" size={ 48 } />
+			</div>
+			<h3 className="rewind-flow__title">{ translate( 'Currently restoring your site' ) }</h3>
+			<ProgressBar percent={ inProgressRewindPercentComplete } />
+			<p className="rewind-flow__info">
+				{ translate(
+					'We are restoring your site back to {{strong}}%(restoreTimestamp)s{{/strong}}.',
+					{
+						args: {
+							restoreTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<CheckYourEmail
+				message={ translate(
+					"Don't want to wait? For your convenience, we'll email you when your site has been fully restored."
+				) }
+			/>
+		</>
+	);
+
+	const renderReady = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-success.svg"
+					alt="jetpack cloud download success"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">
+				{ translate( 'Your site has been successfully restored.' ) }
+			</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'All of your selected files are now restored back to {{strong}}%(restoreTimestamp)s{{/strong}}.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **6** times:
`translate( 'All of your selected files are now restored back to {{strong}}%(restoreTimestamp)s{{/strong}}' )` *ES Score: 9*